Modules/User/Resources/assets/js/app.js

```diff
@@ -7,5 +7,33 @@
 
 
 $(document).ready(function() {
+   
+
+
+  $("#addPancard").click(function(){
+    document.getElementById("demo").innerHTML="Upload Your Pan Card";
+    var data = "<input type='file' name=pancard id='pan'><br><br> <input type='submit' class='btn btn-primary' value='submit'>";
+    document.getElementById('formelment').innerHTML=data;
+
+  })
+  
+  $("#addAadhar").click(function(){
+    document.getElementById("demo").innerHTML="Upload Your Aadhar Card";
+    var data = "<input type='file' name='aadhar' id='aadhar'><br><br> <input type='submit' class='btn btn-primary' value='submit'>";
```

Comment:
Update it at other places as well
```suggestion
    var data="<input type='file' name='aadhar' id='aadhar'><br><br> <input type='submit' class='btn btn-primary' value='submit'>";
```